src/main/java/de/atextor/turtle/formatter/FormattingStyle.java

```diff
@@ -210,7 +212,7 @@ public class FormattingStyle {
     );
 
     @Builder.Default
-    public BiFunction<Resource, Integer, String> anonymousNodeIdGenerator = ( resource, integer ) -> "_:gen" + integer;
+    public BiFunction<Resource, Integer, String> anonymousNodeIdGenerator = ( resource, integer ) -> "gen" + integer;
```

Comment:
This is a good change. Please also update in the README the default value of the description for `anonymousNodeIdGenerator` (where the `_:` is still included).